src/components/Stepper/Step/Illustration.js

```diff
@@ -9,36 +10,33 @@ import {
 
 import signRequestSuccessIllustration from '../../../assets/signRequestSuccess.svg'
 import signRequestFailIllustration from '../../../assets/signRequestFail.svg'
-import signPrompIllustration from '../../../assets/honey.svg'
+import trxBeingMinedIllustration from '../../../assets/honey.svg'
 
 const illustrations = {
-  [STEP_PROMPTING]: signPrompIllustration,
-  [STEP_WORKING]: signRequestSuccessIllustration,
+  [STEP_WORKING]: trxBeingMinedIllustration,
   [STEP_SUCCESS]: signRequestSuccessIllustration,
   [STEP_ERROR]: signRequestFailIllustration,
 }
 
 function Illustration({ status, index }) {
+  const theme = useTheme()
   return (
     <>
-      {/* {status === STEP_WORKING ? (
+      {status === STEP_PROMPTING ? (
```

Comment:
Might make more sense to remove this as it was before and reuse this with the div on the `StatusVisual` component and change the `renderIllustration` logic?

Reply:
i thought about that but i ended thinking that makes more sense to have it in the component in charge of rendering the illustration for each status although is not a illustration itself but act as one

Reply:
Ok lets leave it as is